### PR TITLE
Document how to enable logging of HTTP request and response bodies

### PR DESCRIPTION
**Pull request by scottaddie**
When customers open issues in the Azure SDK for .NET repo, we often ask them to enable logging. It's the HTTP request and response bodies that we're interested in seeing. While helping a customer in https://github.com/Azure/azure-sdk-for-net/issues/29707, I realized the instructions to enable this behavior were missing from the logging doc.

[Internal review page](https://review.docs.microsoft.com/dotnet/azure/sdk/logging?branch=pr-en-us-30312#log-http-request-and-response-bodies)

### CODE REVIEW

**Review by CamSoper — APPROVED** [review]
LGTM

File: docs/azure/sdk/logging.md [hunk 1]
```diff
@@ -202,6 +195,34 @@ Since the `Logging:LogLevel:Azure.Core` key is set to `Debug`, Azure Core librar
...
+    var options = new SecretClientOptions
+    {
+        Diagnostics = 
+        {
+            IsLoggingContentEnabled = true,
+        }
+    };
+    ```
+
+1. Set an [event level](/dotnet/api/system.diagnostics.tracing.eventlevel) of either `Verbose` or `LogAlways`. For example:
```
Comment (JoshLove-msft) [inline docs/azure/sdk/logging.md:219]: This relates to the header about using built-in methods for logging. I wonder if we should keep this section neutral in terms of how users are doing the logging. Regardless of whether they use one of the built-in methods, do custom logging by creating their own AzureEventSourceListener instance, or use appsettings.json in an ASP.NET Core app, they still need to set IsLoggingContentEnabled to true, and they need to have a log level of verbose/debug or higher.

**Review by JoshLove-msft — APPROVED** [review]

**Comment by CamSoper** [comment]
@scottaddie Good to merge?

**Comment by CamSoper** [comment]
Nevermind. 😆

**Pull request merged**